tests/ctst/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "artesca-ctst",
+  "version": "1.0.0",
```

Comment:
Please note that with the way we dynamically get the CTST version from this file, you will have to use 0.2.0, I guess, depending on the version you want to use after merging the Kafke module.

Reply:
See https://github.com/scality/Zenko/pull/1727/files#diff-a8a4d241ff23517544a6ae0ddf6d27875543114739b9ba34b36fd0c7f270e78aR9